CONTRIBUTING.md

```diff
@@ -93,7 +84,7 @@ Please open a dedicated branch for the change in isolation so we can discuss the
 
 Please broadcast the proposal in chat to maximise visibility and the opportunity for everyone to respond.
 
-It is NOT OK to change the behaviour of tests/CI in otherwise unrelated PRs. SOMETIMES it MAY be OK to change CI in a related PR, e.g. adding a new lib that your code requires. DO expect that a change like this will probably attract additional scrutiny during the PR review process, which is unfortunate but important.
+It is not(!) OK to change the behaviour of tests/CI in otherwise unrelated PRs. Sometimes(!) it MAY be OK to change CI in a related PR, e.g. adding a new lib that your code requires. DO expect that a change like this will probably attract additional scrutiny during the PR review process, which is unfortunate but important.
```

Comment:
skim readers often miss the word "not" so i like to capitalise it, the other caps in this paragraph can probably be sentence cased :)